docs/2.develop/contracts/storage.md

```diff
@@ -37,10 +41,17 @@ You can store constants and define contract's attributes.
 ---
 
 ## Data Structures
+Our SDK exposes a series of structures ([Vectors](#vector), [Sets](#set), [Maps](#map) and [Trees](#tree))
+to handle storing data in an efficient way.
 
-Our SDK exposes a series of data structures to simplify handling and storing data. 
+#### Data Prefix
+All structures need to be initialized using a `prefix`, which will be used to identify the structure's keys
+in the [serialized state](./serialization.md#borsh-state-serialization)
 
-The most common ones are [Vectors](#vector), [Sets](#set), [Maps](#map) and [Trees](#tree).
+:::tip
+Always prefer to use the SDK collections over native types in the contract's state, since they are optimized
+for reading and writing into the [serialized key-value storage](./serialization.md#borsh-state-serialization).
+:::
 
 :::caution
 Use **unique IDs** when initializing structures, otherwise they will point to the same key-value references.
```

Comment:
I feel this might need a little more explanation. I can imagine how a web2 dev would think, "oh yeah of course I will add unique key-value pairs to my map!". But this actually means the string when the collection is instantiated in the contract code. 

```suggestion
Use **unique IDs** when initializing structures, otherwise they will point to the same key-value references. Such as: `collectionMap = new UnorderedMap("<my-unique-collection-id>");`
```

Reply:
Good point, indeed it reflects a deeper problem in the article.  I changed the structure of the document to better reflect this.